.github/workflows/test-project-sync.yml

```
@@ -0,0 +1,43 @@
+name: Test Project Sync
+
+on:
+  push:
+    paths:
+      - 'scripts/project-sync/**'           # fix both occurrences
+      - '.github/workflows/test-project-sync.yml'
+  pull_request:
+    paths:
+      - 'scripts/project-sync/**'           # fix both occurrences
+      - '.github/workflows/test-project-sync.yml'
+    concurrency:
```

actionlint (1.7.12)

[error] unexpected key "concurrency" for "pull_request" section. expected one of "branches", "branches-ignore", "paths", "paths-ignore", "tags", "tags-ignore", "types", "workflows"

(syntax-check)